src/Handler/MicrosoftTeamsRecord.php

```diff
@@ -216,14 +216,14 @@ public function setSections(array $record): self
     }
 
     /**
-     * @param mixed $name
+     * @param string|int $name
      * @param mixed $value
      * @param bool $isQuoted
      * @return array
      */
-    public function getFact($name, $value, bool $isQuoted = false): array
+    public function getFact(string|int $name, $value, bool $isQuoted = false): array
```

Comment:
`string|int` is only allowed for PHP 8.1+ in the type definition.

Reply:
Okay, I removed the type declaration.